connectors/src/logger/withlogging.ts

```diff
@@ -74,14 +74,16 @@ export const withLogging = (handler: any) => {
 export function apiError(
   req: Request,
   res: Response,
-  error: APIErrorWithStatusCode
+  apiError: APIErrorWithStatusCode,
+  error?: Error
```

Comment:
NIT: What do you think about being extra explicit and name it `rawError`? I feel it helps understand why we have two fields for errors. Not sure it's wise so feel free to discard!

Reply:
@PopDaph thank you for the review. I love being explicit, but I think error and err are so standard that they do the job.
For example, the logger (pino) actually calls it `err`, and offers in its documentation to call it `error`.

Reply:
If you actually feel strongly about it LMK, otherwise I would vote to keep it as "error". 🙇‍♂️